sdk/eventhub/event-hubs/samples/browserSample/README.md

```diff
@@ -37,6 +37,8 @@ Register a new application in AAD and assign the "Azure Event Hubs Data Owner" r
 
 Ensure your app registration has been configured properly to allow the [implicit grant flow][implicitgrantflow]
```

Comment:
Side random comment: We wont need to do this implicit grant flow once https://github.com/Azure/azure-sdk-for-js/issues/13155 is done because we will be using msal 2.0

Yay!

cc @sadasant, @maorleger